### ISSUE TITLE
bug: Can't scroll up or down while editing

### ISSUE THREAD

**Issue by Kaamkiya** [tauraamui/lilly#238]
To reproduce:

* Open a file
* Press <kbd>i</kbd> to enter editing mode
* Try scrolling up or down

**Comment by tauraamui** [comment]
When you say "scroll" I presume you mean with your mousewheel? That's a missing feature so far rather than a bug. :)

**Comment by Kaamkiya** [comment]
No, I can't scroll with the arrow keys while in `INSERT` mode. It does work while in `NORMAL`, though.

**Comment by tauraamui** [comment]
This issue will be automatically resolved once the migration from line buffer document data structure to using the gap buffer as the document backing structure.
